talisker/gunicorn.py

```diff
@@ -170,6 +182,8 @@ def init(self, parser, opts, args):
             'loglevel': 'DEBUG',
         })
 
+        cfg['worker_exit'] = prometheus_multiprocess_worker_exit
```

Comment:
`workers` is still not known at this point, that's why the envvar check was moved into `load_config()`; and the envvar is going to be set anyways... So I'm just checking if `prometheus_client` is installed.